entity-types/infra-mysqlnode/golden_metrics.yml

```diff
@@ -12,6 +12,9 @@ queriesPerSecond:
       from: MysqlSample
       eventId: entityGuid
       eventName: entityName
+    opentelemetry:
+      select: average(`mysql.operations`)
```

Comment:
I think this is an [dimension](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/0c50ef5a58683a214a509ee15c75279a6334d9b9/receiver/mysqlreceiver/metadata.yaml#L62), not a metric